static/js/publisher/metrics/tooltips.js

```diff
@@ -75,32 +85,38 @@ function snapcraftGraphTooltip(options, data) {
 /**
  *
  * @param {HTMLElement} graphHolder The window offset of the graphs holder.
- * @param {Object} data The  point data.
- * @param {Number} width 
- * @param {Number} height 
+ * @param {Object} data The point data.
+ * @param {Number} width
+ * @param {Number} height
  * @param {HTMLElement} element The tooltip event target element.
- * @returns {Object} Left and top offset of the tooltip.  
+ * @returns {Object} Left and top offset of the tooltip.
  */
 function positionTooltip(graphHolder, data, width, height, element) {
   const tooltipHalfWidth = graphHolder
     .querySelector('.p-tooltip__message')
     .clientWidth / 2;
   const elementHalfWidth = parseFloat(element.getAttribute('width')) / 2;
   const elementSixthHeight = parseFloat(element.getAttribute('height')) / 6;
-  let leftModifier = -26;
+  let leftModifier = -46;
```

Comment:
Magic number changing.... Could you add a little comment to explain that number?

Reply:
Done